SPECS/cloud-init-vmware-guestinfo/cloud-init-vmware-guestinfo.spec

```diff
@@ -0,0 +1,41 @@
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+Name:           cloud-init-vmware-guestinfo
+Version:        1.3.1
+Release:        1
+Summary:        A cloud-init datasource for VMware
+Group:          System/Management
+License:        Apache-2.0
```

Comment:
Please adjust to match our convention of using Fedora short names for the License field. I believe that would be "ASL 2.0" for this license.